app/models/reporting/decisions.rb

```diff
@@ -42,6 +42,10 @@ class Reporting::Decisions < ApplicationRecord
     where(program_type: limit)
   end
 
+  scope :program, ->(limit) do
+    where(program_name: Program.find(limit).pluck(:name))
```

Comment:
Hmm, so, I'm not super excited about matching on program name, though I suppose you can't tell the difference between two identically named programs in the drop down. Why are you plucking here?  Why not?
```suggestion
    where(program_name: Program.where(id: limit).select(:name))
```

Reply:
Reporting::Decisions is storing the program name and not the id. I agree filtering by id would be much better, but I didn't see a great way to get there with the current table setup. 
I can switch this over to a select in the next commmit.